modules/thermo-cycler/thermo-cycler-arduino/gcode.h

```diff
@@ -0,0 +1,83 @@
+#ifndef GCODE_H
+#define GCODE_H
+
+#include "Arduino.h"
+#include <cerrno>
```

Comment:
Consider whether these includes have to be in the header or can live in `gcode.cpp`. If possible, they shouldn't live here.